### DISCUSSION TITLE
Ability to remove programs from the mixer list

### DISCUSSION THREAD

**Discussion by Jeremy2131** [radj307/volume-control#66] [Ideas & Feature Discussions]
Hello, huge thanks for your program
By my suggestion I mean can we be able to remove unnecessary programs from the mixer tab 
I have for example 2 discord instances, telegram, sharex and viber which I would never volume control and between them sometimes I can find my games or chrome so I need to do additional presses on my hotkey to move to that program, it's like just saving few seconds but..

**Comment by radj307** [comment]
This feature request was implemented in version 6.0.0
